src/navigation/navigationstate.cpp

```diff
@@ -33,6 +33,22 @@ namespace {
 
     constexpr double Epsilon = 1E-7;
 
+    // A NavigationState is an object describing an exact camera position and rotation,
+    // in a certain reference frame (per default, the one of the specified Anchor node).
+    // It can be used to set the same camera position at a later point in time, or
+    // navigating to a specific camera position usign the pathnavigation system.
+    //
+    // The camera rotation is specified using Euler angles, in radians. It is also
+    // possible to specify a node to be used as Aim, but note that this will not affect
```

Comment:
No, either way, the aim node does not affect the position or rotation of the camera: that is computed based on the reference frame. 

It just keeps track of what node was the aim when the navigation state is stored, so that it can be set when the navigation state is applied

Reply:
Although, maybe this is unnecessary information?  Might confuse more than it explains

Reply:
Maybe putting in a `\sa` statement to point at the place where to set the aim?

Reply:
What do you mean by that @alexanderbock ?  I don't quite understand

Reply:
ping @alexanderbock :)

Reply:
Nevermind, going through it I misread the comment this referred to. I read "It is also possible to specify a node to be used as Aim" as a pointer to being able to specify it _somewhere else_ and wanted a pointer to where that is done.  But I see now hat it is of course part of the struct